soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -13316,14 +13192,14 @@ void func_8084F104(Player* this, GlobalContext* globalCtx) {
             func_80853148(globalCtx, targetActor);
         }
         else {
-            GetItemEntry* giEntry = &sGetItemTable[D_80854528[this->exchangeItemId - 1] - 1];
+            GetItemEntry giEntry = ItemTable_Retrieve(D_80854528[this->exchangeItemId - 1] - 1);
```

Comment:
i'm thinking replacing `D_80854528[this->exchangeItemId - 1]` is out of scope of this PR, but the other `-1` here fits the rest of the pattern

Reply:
True, yeah haven't touched the exchange items at all, they all behave identically in vanilla and rando anyway.

Reply:
Addressed by 69086f58c2b6bb00a273eefed4744ad456bba6ba

Reply:
made an issue for the other part just in case we ever want to do some exchange item stuff in the future https://github.com/HarbourMasters/Shipwright/issues/1175